clusters/build-clusters/02_cluster/openshift-apiserver/README.md

```diff
@@ -0,0 +1,48 @@
+# openshift-ingress-operator
+
+## CA Certificate for the API servers
+
+Openshift 4.2 has doc on [this topic](https://docs.openshift.com/container-platform/4.2/authentication/certificates/api-server.html).
+
+### Manual steps
+
+#### AWS IAM configuration
+
+[set up aws IAM user](https://github.com/Neilpang/acme.sh/wiki/How-to-use-Amazon-Route53-API): user `openshift-ci-robot` in group `acme` which
+    has the policy `acme`.
+
+#### Generate the certificate
+
+Using [acme.sh](https://github.com/Neilpang/acme.sh)
+
+Generate certificates by cert-manager:
+
+```
+$ oc --as system:admin --context build02 apply -f clusters/build-clusters/02_cluster/cert-manager/apiserver-build02_certificate.yaml
```

Comment:
Is this needed? Don't they get auto applied?

Reply:
Updated with clarification.